docs/src/user_guide/step4_running.md

```diff
@@ -1,4 +1,26 @@
-# Command Line Interface
+# [Step 4: Running a simulation](@id run_wflow)
+
+## Using Julia
```

Comment:
With Julia install you can also use the command line interface, you don’t necessarily need to write a script: julia -e "using Wflow; Wflow.run()" "path/to/config.toml"

Reply:
That's correct, overlooked this option. It is now added added as an option as well!